app/models/channel.rb

```diff
@@ -1,14 +1,21 @@
 class Channel < ActiveRecord::Base
-  attr_accessible :category, :content, :last_verified, :name, :person_id, :usage, :status
 
-  belongs_to :person
 
+  attr_accessible :category, :content, :name, :status, :usage, :carrier, :sms_available, :priority, :type, :_destroy, :person_id
```

Comment:
Please break this into two lines. Most companies have style guides that prohibit lines this long